bugbug/repository.py

```diff
@@ -130,15 +133,26 @@ def download_commits(repo_dir, date_from):
 
     # Total previous number of commits by the author.
     total_commits_by_author = defaultdict(int)
+    total_reviews_by_reviewer = defaultdict(int)
     # Previous commits by the author, in a 90 days window.
     commits_by_author = defaultdict(list)
 
     global author_experience
+    global reviewer_experience
     global author_experience_90_days
+
     for commit in commits:
         author_experience[commit] = total_commits_by_author[commit.author]
+
         total_commits_by_author[commit.author] += 1
 
+        if commit.reviewers != ('',):
```

Comment:
When there are no reviewers, what is `rev[6]`? Could we make it so that commit.reviewers is empty when there are no reviewers, instead of being a tuple with only the '' element?

Reply:
rev[6] is `b' '` when there are no reviewers. Sure, that can be done.